docs/advanced-topics/checker-workflows.rst

```diff
@@ -120,43 +122,56 @@ is important that the checker workflow produce consistent exit codes. We
 require using an exit code of 0 for success and an exit code of not 0
 for failures.
 
-We also recommend producing the following two output files containing
-the stdout and stderr respectively:
-
-* log.stdout
-* log.stderr
+We also recommend producing the stderr and stdout logs. The exact filenames
```

Comment:
Was like this, it says to produce stderr and stdout logs, but then it tells where the platforms produce them. Doesn't sound quite right to me.